compiler/rustc_lint/messages.ftl

```diff
@@ -403,8 +403,9 @@ lint_inner_macro_attribute_unstable = inner macro attributes are unstable
 
 lint_invalid_asm_label_binary = avoid using labels containing only the digits `0` and `1` in inline assembly
     .label = use a different label that doesn't start with `0` or `1`
-    .note = an LLVM bug makes these labels ambiguous with a binary literal number
-    .note = see <https://bugs.llvm.org/show_bug.cgi?id=36144> for more information
+    .help = start numbering with `2` instead
+    .note1 = an LLVM bug makes these labels ambiguous with a binary literal number on x86
+    .note2 = see <https://github.com/llvm/llvm-project/issues/99547> for more information
```

Comment:
nit: it isn't necessary for both of them to use a different identifier, only of them needs to, the other one can stay as always (`.note = ...` and `#[note]`)

Reply:
I just mirrored a few lines up in https://github.com/rust-lang/rust/pull/127935/files#diff-4a2362ab93352b636e03b023fecbe61e96c3a07f1be1393d67bbd87d87374f50L2060-L2063

Seems like we should have a way to detect unused `.note`s somehow, since the second wasn't being used until this change.

Reply:
> Seems like we should have a way to detect unused labels, since the second .note wasn't being used until this change.

We should indeed.

Anyway, no need to change anything. Just wanted you to be aware of this.

Reply:
Looks like it's at least been considered https://github.com/rust-lang/rust/issues/110754